### PR TITLE
add info about free space

### PR DESCRIPTION
**Pull request by VLG17**
https://github.com/MicrosoftDocs/OfficeDocs-Support/issues/1025

### CODE REVIEW

**Comment by helenclu** [comment]
Better provide more information about the following:
- How to decide if this is a DB issue.
- How to resolve the issue.

**Comment by VLG17** [comment]
> Better provide more information about the following:
> 
> * How to decide if this is a DB issue.
> * How to resolve the issue.

@helenclu It's not a DB issue, it's the fact that the Sharepoint server which hosts the SharePoint DB has run out of space. Do we need to add steps describing how to check that? You would be able to see that directly looking at the storage of the drive that holds that DB.

Not sure what else I should add here.

**Comment by helenclu** [comment]
@VLG17 , pls check the following blogs:
http://mehsanullah.blogspot.com/2017/04/sharepoint-2013-error-sorry-something.html
http://melick-rajee.blogspot.com/2017/08/the-url-is-invalid-it-may-refer-to.html

To address the customer issue, just adding the cause doesn't help much.

**Review by VLG17 — COMMENTED** [review]
added more info as advised by helenclu

File: SharePoint/SharePointOnline/lists-and-libraries/url-invalid-upload-file-to-list.md [hunk 1]
```diff
@@ -43,6 +43,20 @@ To work around this issue, remove the Version column from the list of Indexed Co
 
 1. Upload any files that you previously experienced the issue with.
 
+
+If the above solution does not solve the issue, please also check that the Sharepoint Database Server is not running out of free space.
```
Comment (JohanFreelancer9) [inline SharePoint/SharePointOnline/lists-and-libraries/url-invalid-upload-file-to-list.md:47]:
```suggestion
If the previous solution does not solve the issue, also check that the Sharepoint Database Server is not running out of free space.
```

File: SharePoint/SharePointOnline/lists-and-libraries/url-invalid-upload-file-to-list.md [hunk 2]
```diff
@@ -43,6 +43,20 @@ To work around this issue, remove the Version column from the list of Indexed Co
...
+
+1. Go to the SharePoint application server, open Event Viewer and see the details of the actual error. If it's related to database(s) running out of space, proceed to the next step.
```
Comment (JohanFreelancer9) [inline SharePoint/SharePointOnline/lists-and-libraries/url-invalid-upload-file-to-list.md:49]:
```suggestion
1. Go to the SharePoint application server, open Event Viewer, and see the details of the actual error. If it's related to database(s) running out of space, proceed to the next step.
```

File: SharePoint/SharePointOnline/lists-and-libraries/url-invalid-upload-file-to-list.md [hunk 3]
```diff
@@ -43,6 +43,20 @@ To work around this issue, remove the Version column from the list of Indexed Co
...
+
+1. Check that the following SharePoint databases have enough space to perform actions, and if not, please check the size of the database log and configure the log storage limit accordingly.
```
Comment (JohanFreelancer9) [inline SharePoint/SharePointOnline/lists-and-libraries/url-invalid-upload-file-to-list.md:51]:
```suggestion
1. Check that the following SharePoint databases have enough space to perform actions, and if not, check the size of the database log and configure the log storage limit accordingly. 
```

File: SharePoint/SharePointOnline/lists-and-libraries/url-invalid-upload-file-to-list.md [hunk 4]
```diff
@@ -43,6 +43,20 @@ To work around this issue, remove the Version column from the list of Indexed Co
...
 1. Upload any files that you previously experienced the issue with.
 
+
+If the above solution does not solve the issue, please also check that the Sharepoint Database Server is not running out of free space.
+
+1. Go to the SharePoint application server, open Event Viewer and see the details of the actual error. If it's related to database(s) running out of space, proceed to the next step.
+
+1. Check that the following SharePoint databases have enough space to perform actions, and if not, please check the size of the database log and configure the log storage limit accordingly. 
+
+SharePoing Content Database
```
Comment (JohanFreelancer9) [inline SharePoint/SharePointOnline/lists-and-libraries/url-invalid-upload-file-to-list.md:53]:
```suggestion
SharePoint content database
```

File: SharePoint/SharePointOnline/lists-and-libraries/url-invalid-upload-file-to-list.md [hunk 5]
```diff
@@ -43,6 +43,20 @@ To work around this issue, remove the Version column from the list of Indexed Co
...
+SharePoint Contect Database Logs
```
Comment (JohanFreelancer9) [inline SharePoint/SharePointOnline/lists-and-libraries/url-invalid-upload-file-to-list.md:54]:
```suggestion
SharePoint content database logs
```

File: SharePoint/SharePointOnline/lists-and-libraries/url-invalid-upload-file-to-list.md [hunk 6]
```diff
@@ -43,6 +43,20 @@ To work around this issue, remove the Version column from the list of Indexed Co
...
+Temp Database Logs
```
Comment (JohanFreelancer9) [inline SharePoint/SharePointOnline/lists-and-libraries/url-invalid-upload-file-to-list.md:55]:
```suggestion
Temp database logs
```

File: SharePoint/SharePointOnline/lists-and-libraries/url-invalid-upload-file-to-list.md [hunk 7]
```diff
@@ -43,6 +43,20 @@ To work around this issue, remove the Version column from the list of Indexed Co
...
+SharePoint Service Databases
```
Comment (JohanFreelancer9) [inline SharePoint/SharePointOnline/lists-and-libraries/url-invalid-upload-file-to-list.md:56]:
```suggestion
SharePoint service databases
```

File: SharePoint/SharePointOnline/lists-and-libraries/url-invalid-upload-file-to-list.md [hunk 8]
```diff
@@ -43,6 +43,20 @@ To work around this issue, remove the Version column from the list of Indexed Co
...
+SharePoint Service Databases Logs
```
Comment (JohanFreelancer9) [inline SharePoint/SharePointOnline/lists-and-libraries/url-invalid-upload-file-to-list.md:57]:
```suggestion
SharePoint service databases logs
```

File: SharePoint/SharePointOnline/lists-and-libraries/url-invalid-upload-file-to-list.md [hunk 9]
```diff
@@ -43,6 +43,20 @@ To work around this issue, remove the Version column from the list of Indexed Co
...
+SharePoint Service Databases Temp Logs
```
Comment (JohanFreelancer9) [inline SharePoint/SharePointOnline/lists-and-libraries/url-invalid-upload-file-to-list.md:58]:
```suggestion
SharePoint service databases temp logs
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by simonxjx** [comment]
Checking.

**Comment by VLG17** [comment]
anything else needed from my side?

**Pull request closed without merging**